media/css/mozorg/home/home.scss

```diff
@@ -472,7 +472,7 @@ main {
         padding-top: 40px;
 
         &:before {
-            @include at2x('/media/img/firefox/common/firefox-logo.png');
+            @include at2x('/media/img/firefox/template/logo.png');
```

Comment:
at first I completely ignored the fact that line of code could a mixin lol. After you mentioned it, I got back & checked out the at2x.scss file. So yeah, now I feel pretty stupid doing it :stuck_out_tongue: